### PR TITLE
ADEN-3994 calc the page height

### PR DESCRIPTION
**Pull request by jacekjursza**
Replace rrspace parameter (in GPT page level params) with ahspace (article height).

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [desktop.run.js#L22](https://github.com/Wikia/app/blob/713559e84583077effb82703a68f3405c0ad80d5/extensions/wikia/AdEngine/js/run/desktop.run.js#L22): Function has 18 parameters which is greater than 7 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. MAJOR [desktop.run.js#L103](https://github.com/Wikia/app/blob/713559e84583077effb82703a68f3405c0ad80d5/extensions/wikia/AdEngine/js/run/desktop.run.js#L103): Function has 10 parameters which is greater than 7 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)

File: extensions/wikia/AdEngine/js/run/desktop.run.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ require([
 	// Everything starts after content and JS
 	win.wgAfterContentAndJS.push(function () {
 		if (floatingRail) {
-			pageLevelParams.add('rrspace', floatingRail.getAvailableSpaceParameter().toString());
+			pageLevelParams.add('ahspace', floatingRail.getArticleHeightParameter().toString());
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/run/desktop.run.js]: What do you think about moving it to https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/AdLogicPageParams.js#L164 - will it be useful info on Mercury?

additional: s/ahspace/ah

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [desktop.run.js#L22](https://github.com/Wikia/app/blob/c97c089cd2ca2544df3c5f1530ca4f89769372fb/extensions/wikia/AdEngine/js/run/desktop.run.js#L22): Function has 18 parameters which is greater than 7 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)
2. MAJOR [desktop.run.js#L103](https://github.com/Wikia/app/blob/c97c089cd2ca2544df3c5f1530ca4f89769372fb/extensions/wikia/AdEngine/js/run/desktop.run.js#L103): Function has 10 parameters which is greater than 7 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)

File: extensions/wikia/AdEngine/js/run/desktop.run.js (hunk 1 above)
Comment (fraszczakszymon) [reply extensions/wikia/AdEngine/js/run/desktop.run.js]: Discussed offline

**Comment by fraszczakszymon** [comment]
LGTM 👍

**Comment by bulaszenko** [comment]
Tested works good

**Pull request merged**